R/guide-colorsteps.R

```diff
@@ -114,7 +114,11 @@ GuideColoursteps <- ggproto(
     } else {
       key$.value <- breaks
     }
-    key$.label <- scale$get_labels(breaks)
+    labels <- scale$get_labels(breaks)
+    if (is.expression(labels)) {
+      labels <- as.list(labels)
+    }
```

Comment:
Wondering if it wouldn't be better if this was the responsibility of `Scale$get_labels()`.

Reply:
In some places for sure.

Reply:
I think that it logically belongs in `get_labels()` as it ensures the output of that function is usable. It shouldn't be the responsibility of the caller to further process the return value to make it work